src/github.ts

```diff
@@ -121,13 +121,15 @@ export async function createRelease(
   commitish: string,
   body: string,
   draft: boolean,
-  prerelease: boolean
+  prerelease: boolean,
+  discussionCategory?: string
```

Comment:
I know I can't complain too loudly, as it looks like I forgot to update the TSDoc when I added `prerelease` ( :grimacing: ), but it'd be good to keep it as up-to-date as possible.

Reply:
Good point let me leave the campground cleaner then expected!